src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,36 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+        LocalDate startDate = LocalDate.parse(dateFrom, formatter);
+        LocalDate endDate = LocalDate.parse(dateTo, formatter);
+        int[] salaries = new int[names.length];
+        for (String datum : data) {
+            String[] temp = datum.split(" ");
+            if (temp[0].length() < 10) {
+                temp[0] = "0" + temp[0];
+            }
```

Comment:
what is this check for?

Reply:
oh,  i remember it is cheking if input is something like d.mm.yyyy or dd.mm.yyyy

Reply:
i see now tests pass fine without it. it seems redundant.